src/app/apps/auth/components/login/login.component.ts

```diff
@@ -14,22 +15,32 @@ export class LoginComponent implements OnInit {
     public password: string;
     public loading = false;
 
-    constructor(private plex: Plex, private auth: Auth, private router: Router, public ws: WebSocketService) { }
+    constructor(private plex: Plex, private auth: Auth, private router: Router, public ws: WebSocketService, public us: UsuarioService, public ds: DisclaimerService) { }
 
     ngOnInit() {
         this.auth.logout();
         this.ws.close();
     }
 
-
     login(event) {
         if (event.formValid) {
             this.loading = true;
             this.auth.login(this.usuario.toString(), this.password)
                 .subscribe((data) => {
                     this.plex.updateUserInfo({ usuario: this.auth.usuario });
-                    this.ws.setToken(window.sessionStorage.getItem('jwt'));
-                    this.router.navigate(['/auth/select-organizacion']);
+                    this.ds.get({ activo: true }).subscribe(disclaimers => {
+                        if (disclaimers) {
+                            let disclaimer = disclaimers[0];
+                            this.ws.setToken(window.sessionStorage.getItem('jwt'));
+                            this.us.getDisclaimers(this.auth.usuario).subscribe((userDisclaimers) => {
+                                if (userDisclaimers.some(item => item._id === disclaimer.id)) {
```

Comment:
No usar del lado de la app el _id. Para eso se usa el virtual "id" que lo genera mongoose
```suggestion
                                if (userDisclaimers.some(item => item.id === disclaimer.id)) {
```
OJO: Para que esto funcione modificar en la api en la parte de schema de authUser quitar id y poner _id para que genere el virtual correctamente.

Reply:
listo!